### ISSUE TITLE
Events on default bucket do not have consistent payload

### ISSUE THREAD

**Issue by leplatrem** [Kinto/kinto#704]
For `GET /v1/buckets/default/collections/articles/records'`:

```
{'resource_name': 'collection', 'user_id': 'basicauth:3a0c56d278def4113f38d0cfff6db1b06b84fcc4384ee890cf7bbaa772317e10', 'action': 'create', 'timestamp': 1467729134102, 'subpath': u'collections/articles/records', 'uri': u'/buckets/default/collections/articles/records'}
```

* `subpath` instead of `bucket_id` ...
* `uri`
* see https://github.com/Kinto/kinto/blob/3.2.1/kinto/plugins/default_bucket/test_plugin.py#L267-L279 (blame=me)

**Comment by leplatrem** [comment]
Also, the resource_name is the same when bucket and collection are created implicitly in the same request:

```
{'resource_name': 'collection', 'user_id': 'basicauth:3a0c56d278def4113f38d0cfff6db1b06b84fcc4384ee890cf7bbaa772317e10', 'action': 'create', 'timestamp': 1467730856639, 'subpath': u'collections/articles/records', 'uri': u'/buckets/default/collections/articles/records'}
{'resource_name': 'collection', 'user_id': 'basicauth:3a0c56d278def4113f38d0cfff6db1b06b84fcc4384ee890cf7bbaa772317e10', 'action': 'create', 'timestamp': 1467730856639, 'subpath': u'collections/articles/records', 'uri': u'/buckets/default/collections/articles/records'}
```

**Comment by leplatrem** [comment]
Fixed in #705
